[4.1.0] Interceptors for Choreo Connect (#4953)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/interceptor-microservice/interceptor-context-information.md

> # Interceptor Contexts Information

Reviewer: ```suggestion
# Interceptor Context and Invocation Context
```

## Interceptor Context

Interceptor context is a string to string map that can be shared with request flow interceptor and response flow interceptor.

> From the request interceptor you can set it in the response body of the interceptor, and the Choreo Connect Router will add them

Reviewer 2: `set it in the response body of the interceptor`? or the backend?

> From the request interceptor you can set it in the response body of the interceptor, and the Choreo Connect Router will add them

Reviewer 2: ```suggestion
From the request interceptor you can return it in the response body, and the Choreo Connect Router will add them
```

to the request it made to the response interceptor. Following is a sample interceptor context.

```json
{
  "interceptorContext": {
    "key1": "value1"
  }
}
```

## Invocation Context

Invocation context describes the request information. This is an optional object that you can request from

> Choreo Connect Router to your interceptor service by referring it in the `includes` in Open API Definition.

Reviewer 2: ```suggestion
Choreo Connect Router to your interceptor service by referring it in the `includes` in Open API Definition. 
Invocation context has the following information.
```
